project/build.scala

```diff
@@ -50,7 +50,8 @@ object Geoprocessing extends Build {
   )
 
   val resolutionRepos = Seq(
-    "Job Server Bintray" at "https://dl.bintray.com/spark-jobserver/maven"
+    Resolver.bintrayRepo("spark-jobserver", "spark-jobserver"),
```

Comment:
I was able to build and test the assembly JAR without it, so I don't believe that it is necessary. Removed in https://github.com/WikiWatershed/mmw-geoprocessing/commit/3ccc90527efc7529099ced4c05c23a40509b4e7d.